src/dirs.proj

```diff
@@ -1,6 +1,11 @@
 <Project ToolsVersion="14.0" DefaultTargets="BuildAndTest" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="dir.props" />
 
+  <!-- required to build the projects in their specified order -->
+  <PropertyGroup>
+    <SerializeProjects>true</SerializeProjects>
+  </PropertyGroup>
+
   <ItemGroup>
     <Project Include="ref.builds" />
```

Comment:
We should probably make this targets file become a no-op and just include ref.builds and src.builds in build.proj.